### PR TITLE
feat: make website partly multilingual

### PR DESCRIPTION
**Pull request by maelle**
## Summary of the changes

- Made blog posts in not English actual translations so Hugo knows which is a translation of what. Made each post in each language fully in that language: date, "browse by tag" etc all translated. Translated tags too. Example https://deploy-preview-506--ropensci.netlify.app/es/blog/2023/02/28/r-universe-stars-2-es/
- Made code of conduct in Spanish an actual translation too.
- Added redirects for the old blog posts and COC links.
- Added a warning in the navbar on French and Spanish pages. It links to issues labelled "translation" in this repository.
- Created archive and technotes list pages for Spanish and French. Example https://deploy-preview-506--ropensci.netlify.app/es/archive/
- Linked to blog posts in other languages from each post and from the blog main list. Linked to all Spanish posts from the English blog main page, as a highlight. https://deploy-preview-506--ropensci.netlify.app/blog and https://deploy-preview-506--ropensci.netlify.app/es/blog
- Add author pages in other languages = Duplicates of English files for now. We could translate bios. https://deploy-preview-506--ropensci.netlify.app/es/author/, https://deploy-preview-506--ropensci.netlify.app/es/author/yanina-bellini-saibene/
- All links to other languages should have the "lang" attribute.
- The HEAD has the correct lang attribute, but also and most importantly hreflang links to versions of the same page in other languages.

With this PR the English pages are "clean": the links to non English content have a "lang='es'" tag for instance. There's very little text in Spanish (only the highlight on https://deploy-preview-506--ropensci.netlify.app/blog/ that says "some posts are also available in Spanish". On the contrary, the Spanish pages are full of English pieces: the footer and navbar in particular.

However this is an improvement over the current situation.

## Feedback needed on

- anything you find amiss, ugly. Suggestions welcome!
- :warning: *would you find it clearer/better to have a subdomain per language (esp.ropensci.org) as opposed to a folder per language?*
- how to make the warning in the navbar nicer.
- the blog featured cards in particular: do the language links look ok? See below

image

## Feedback not needed on
- the navbar and the footer being in English... we'd translate them in another PR later when we translate more static files.

### CODE REVIEW

**Comment by yabellini** [comment]
Looks really nice Maelle !!

Addin one TODO:
- [ ] Add the language tag to all the pots (the list is missing the one I contributed about r-universe some time ago, for example)

**Comment by maelle** [comment]
>  Add the language tag to all the pots (the list is missing the one I contributed about r-universe some time ago, for example)

which post? btw I think we should remove the "Spanish" tags since they will be listed in a different way, and I think we should translate all tags. let's discuss!

**Comment by yabellini** [comment]
Agree about the translation of the tag! This blog post: https://ropensci.org/blog/2021/09/21/creando-tu-r-universe/

**Comment by maelle** [comment]
oh thanks, that's a post present in Spanish only, let's see whether that breaks anything!

File: config.toml [hunk 1]
```diff
@@ -2,6 +2,23 @@ baseurl = "https://ropensci.org/"
...
 theme = "ropensci"
+
+DefaultContentLanguage = "en"
+[languages]
+  [languages.en]
+    title = "rOpenSci - open tools for open science"
+    languageName = "English"
+    weight = 2
+  [languages.es]
+    title = "rOpenSci - open tools for open science"
```
Comment (maelle) [inline config.toml:13]: @yabellini how would you translate this?

Comment (yabellini) [inline config.toml:13]:
```suggestion
    title = "rOpenSci - herramientas abiertas para una ciencia abierta"
```

File: content/archive/_index.es.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
++++
+title = "Archivo"
+description = "Todas las entradas, por fechas"
```
Comment (yabellini) [inline content/archive/_index.es.md:3]:
```suggestion
description = "Todos los artículos, por fechas"
```

File: content/author/_index.es.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
++++
+date = "2017-07-13T22:00:37-07:00"
+title = "Autores"
```
Comment (yabellini) [inline content/author/_index.es.md:3]:
```suggestion
title = "Autores y autoras"
```

File: content/author/_index.es.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+description = "Autores en el Blog de rOpenSci (español)"
```
Comment (yabellini) [inline content/author/_index.es.md:4]:
```suggestion
description = "Autoras y autores en el Blog de rOpenSci (español)"
```

File: content/author/alejandra-bellini/_index.es.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+name: Alejandra Bellini
+bio: Social Communication. Journalist. Specialization in Political Journalism and Digital Marketing.
```
Comment (yabellini) [inline content/author/alejandra-bellini/_index.es.md:3]:
```suggestion
bio: Comunicadora Social. Periodista. Especialita en periodísmo político y Marketing Digital.  
```

File: content/blog/2021-07-27-censo2017/index.es.md [hunk 6]
```diff
@@ -10,21 +10,18 @@ categories:
...
-  - geospatial
-  - community
-  - software-peer-review
+  - ciencia-abierta
+  - investigación reproducible
+  - acceso a datos
+  - extracción de datos
+  - geoespacial
+  - comunidad
+  - revista software por pares
```
Comment (yabellini) [inline content/blog/2021-07-27-censo2017/index.es.md:19]:
```suggestion
  - revisión de software por pares
```

File: content/blog/2022-04-19-software-review-editorial-challenges/index.es.md [hunk 7]
```diff
@@ -7,8 +7,7 @@ date: '2022-04-19T00:00:01-07:00'
 slug: software-review-editorial-challenges-es
 tags:
 - Software Peer Review
```
Comment (yabellini) [inline content/blog/2022-04-19-software-review-editorial-challenges/index.es.md:9]:
```suggestion
- revisión de software
```

**Review by yabellini — COMMENTED** [review]
Revisiones menores

**Comment by maelle** [comment]
@yabellini ¡muchas gracias!

**Comment by karthik** [comment]
Not in the sidebar, but in the feature card.

**Comment by maelle** [comment]
The feature card does not have the text Other languages in the most recent version: 

image

File: config.toml (hunk 1 above)
Comment (yabellini) [reply config.toml]: Done!

**Comment by jeroen** [comment]
I really like this. 

Perhaps one idea to make the translation links more intuitive, if you add the current language to the list, and make that bold. So maybe something like this:

Screenshot 2023-03-23 at 5 12 00 PM

Or indeed perhaps even better would be a dropdown element (if we can style it nicely) so it is intuitive what is the current value, and that it can be changed.

**Comment by jeroen** [comment]
🎉

**Pull request merged**